Update README.md file (#408)
tendermint/liquidity: README.md

## Development

### Test

```bash
$ make test-all
```

### 1\. Setup local testnet using script

```bash
# This script bootstraps a single local testnet.
# Note that config, data, and keys are created in the ./data/localnet folder and
# RPC, GRPC, and REST ports are all open.
$ make localnet
```

### 1.1 Broadcast transactions using CLI command-line interface

> Some sample scripts are available in [scripts](https://github.com/tendermint/liquidity/tree/develop/scripts) folder, which will help you to test out the liquidity module interface.

Reviewer: ```suggestion
Sample scripts are provided in [scripts](https://github.com/tendermint/liquidity/tree/develop/scripts) folder to help you to test the liquidity module interface.
```

# Encode the signed tx
liquidityd tx encode tx_swap_signed.json
```

> ### 2.2 Broadcaste transactions using REST APIs

Reviewer: ```suggestion
### 2.2 Broadcast transactions using REST APIs
```

Author: Oops! Great catch

> An example of broadcasting transactions using REST API (via gRPC-gateway) can be found in this [link](https://github.com/cosmos/cosmos-sdk/blob/master/docs/migrations/rest.md#migrating-to-new-rest-endpoints). Note that API server should be enabled in `$HOME/.liquidityapp/config/app.toml` to test this.

Reviewer: ```suggestion
For an example of broadcasting transactions using REST API (via gRPC-gateway), see Cosmos SDK [Migrating to New REST Endpoints](https://github.com/cosmos/cosmos-sdk/blob/master/docs/migrations/rest.md#migrating-to-new-rest-endpoints). Testing requires that the API server is enabled in `$HOME/.liquidityapp/config/app.toml`.
```

Reviewer: instead of "link" let's use descriptive phrases that provide context for the material that we're linking to, per https://developers.google.com/style/link-text

### Protobuf and Swagger

The API documentation for the liquidity module is available on `http://localhost:1317/swagger-liquidity/` after you successfully boostrap a testnet in your local computer.

> Note that `swagger` config must be `true` in `$HOME/.liquidityapp/config/app.toml`. The public Swagger API docs are also available on [Cosmos SDK Liquidity Module - REST and gRPC Gateway docs](https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs).

Reviewer: ```suggestion
You must set `swagger` config to `true` in `$HOME/.liquidityapp/config/app.toml`. The public Swagger API docs are also available on [Cosmos SDK Liquidity Module - REST and gRPC Gateway docs](https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs).
```

# Generate `swagger.yaml` from `proto/*.proto`
$ make proto-swagger-gen
```

## Resources

> To dive into more about the liquidity module, check out the following resources.

Reviewer: ```suggestion
To learn more about the liquidity module, check out the following resources:
```
